
int32 img_x
int32 img_y
---------
float64 pcd_x
float64 pcd_y
float64 pcd_z

